exercises/tom_clayton/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,158 @@
+/**
+ * Import statements
+ */
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <string.h>
+#include <vector>
+#include <map>
+using namespace std;
+
+/**
+ * Function to return a string filepath from user input
+ */
+
+std::string getInputFn() {
+  std::cout << "Please enter the path of the file to be analysed" << std::endl;
+  std::string fin;
+  std::cin >> fin;
+  return fin;
+}
+
+/**
+ * Function to split up a string and return a vector of words with
+ * len <4, and converted to lower case
+ */
+
+std::vector<std::string> splitByDelim(const std::string& s, char delim) {
+  
+  std::vector<std::string> tokens;
+  std::string token;
+  std::istringstream tokenStream(s);
+  
+  while (std::getline(tokenStream, token, delim)) {
+    if(token.size() < 4) {
+      continue;
+    }
+
+    std::transform(token.begin(), token.end(), token.begin(),
+    [](unsigned char c){ return std::tolower(c);});
+    
+    tokens.push_back(token);
+    //cout << token << endl;
```

Comment:
Please can you delete this. It's not unusual to have commented out stuff like this while developing but we try to remove this kind of thing before merging the code into master

Reply:
Done